### PR TITLE
Prevent line lengths over 120 from throwing errors

### PR DESCRIPTION
**Pull request by JoshuaLuckers**
### What does it do?
- Overrides the hard line-length limit on the PHPCS check
- Disable fail on warnings for `thenabeel/action-phpcs`

### Why is it needed?
Prevent PHPCS from failing code with lines longer than 120 characters.

### How to test
No idea what's the best approach to test this. 

### Related issue(s)/PR(s)
Issue #15790
PR #15797

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
@opengeek I believe this change will do the trick!

**Review by Mark-H — APPROVED** [review]

File: phpcs.xml
Comment (opengeek) [inline phpcs.xml]: I don't think this override is necessary as this is what PSR-12 defines already.

**Review by opengeek — CHANGES_REQUESTED** [review]

File: phpcs.xml
Comment (smg6511) [reply phpcs.xml]: Right, it's really just the fail_on_warnings change that's needed to keep warnings from failing the entire code quality check. That said, as I mentioned in issue #15790 and @opengeek concurred on, we should just disable this code quality step altogether until a defined time period where updating the whole codebase to be PSR-12 compliant is the sole or at least main focus.

Comment (opengeek) [reply phpcs.xml]: Actually, I don't see any reason to disable it. It's not blocking merging as it is not a required check. It's nice to have the information, but we need to make it clear to contributors that they do NOT need to attempt to fix the files they change for now.

Comment (smg6511) [reply phpcs.xml]: I don't know, it's pretty difficult spotting actual content changes if someone has gone to the effort of fixing all the formatting, especially in larger php files. The trouble is that many of the code checks cause an error, not a warning.  The one particularly problematic one is the casing of class names, which will have to be either carefully changed or a custom sniff would have to replace the default PSR-12 one to allow the current style. Is there a way to allow the errors and warnings to show but not cause failure of the code check? Anyway, just something to consider ;-)

Comment (JoshuaLuckers) [reply phpcs.xml]: If the formatting changes are done in a separate commit it isn't that big of a problem.

Comment (opengeek) [reply phpcs.xml]: Until I squash and merge it isn't. 😛

**Review by opengeek — APPROVED** [review]

**Pull request merged**
